src/main/java/com/plivo/api/models/verify/UpdateVerifiedCallerIdResponse.java

```diff
@@ -0,0 +1,18 @@
+package com.plivo.api.models.verify;
+
+import com.plivo.api.models.base.BaseResponse;
+
+import java.util.Date;
+
+public class UpdateVerifiedCallerIdResponse extends BaseResponse {
+
+  private String alias;
```

Comment:
update, get and getall and validate have all the same base response. Can't we create one base response and use that?